pallets/proposals/src/lib.rs

```diff
@@ -692,6 +730,13 @@ impl<BlockNumber: From<u32>> Round<BlockNumber> {
     }
 }
 
+/// <HB SBP Review:
+///
+/// I would recommend to consider library sp_arithmetic for all what it is percentage related: https://docs.rs/sp-arithmetic/15.0.0/sp_arithmetic/per_things/index.html
```

Comment:
@cuteolaf 
Can we do this? 
Use the Percent types in sp_arithmetic instead of using u8s etc